"npm run release" fails

I try to run "npm run release" after running "npm install" in #108 but it fails:

```
$ npm run release

> zonemaster-gui@3.1.0 release /home/matsd/git/zonemaster-gui
> npm run build && node scripts/create_release.js

npm WARN invalid config loglevel="notice"

> zonemaster-gui@3.1.0 build /home/matsd/git/zonemaster-gui
> node scripts/faq.js && ng build --prod

events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open './src/assets/faqs/gui-faq-da.html'
Emitted 'error' event at:
    at lazyFs.open (internal/fs/streams.js:277:12)
    at FSReqWrap.args [as oncomplete] (fs.js:140:20)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zonemaster-gui@3.1.0 build: `node scripts/faq.js && ng build --prod`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the zonemaster-gui@3.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/matsd/.npm/_logs/2020-03-30T06_56_38_089Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zonemaster-gui@3.1.0 release: `npm run build && node scripts/create_release.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the zonemaster-gui@3.1.0 release script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/matsd/.npm/_logs/2020-03-30T06_56_38_214Z-debug.log
```

/home/matsd/.npm/_logs/2020-03-30T06_56_38_214Z-debug.log  says:

```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node', '/usr/bin/npm', 'run', 'release' ]
2 info using npm@6.14.4
3 info using node@v10.19.0
4 verbose run-script [ 'prerelease', 'release', 'postrelease' ]
5 info lifecycle zonemaster-gui@3.1.0~prerelease: zonemaster-gui@3.1.0
6 info lifecycle zonemaster-gui@3.1.0~release: zonemaster-gui@3.1.0
7 verbose lifecycle zonemaster-gui@3.1.0~release: unsafe-perm in lifecycle true
8 verbose lifecycle zonemaster-gui@3.1.0~release: PATH: /usr/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/matsd/git/zonemaster-gui/node_modules/.bin:/home/matsd/bin:/home/matsd/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
9 verbose lifecycle zonemaster-gui@3.1.0~release: CWD: /home/matsd/git/zonemaster-gui
10 silly lifecycle zonemaster-gui@3.1.0~release: Args: [ '-c', 'npm run build && node scripts/create_release.js' ]
11 silly lifecycle zonemaster-gui@3.1.0~release: Returned: code: 1  signal: null
12 info lifecycle zonemaster-gui@3.1.0~release: Failed to exec release script
13 verbose stack Error: zonemaster-gui@3.1.0 release: `npm run build && node scripts/create_release.js`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:198:13)
13 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:198:13)
13 verbose stack     at maybeClose (internal/child_process.js:982:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
14 verbose pkgid zonemaster-gui@3.1.0
15 verbose cwd /home/matsd/git/zonemaster-gui
16 verbose Linux 4.4.0-176-generic
17 verbose argv "/usr/bin/node" "/usr/bin/npm" "run" "release"
18 verbose node v10.19.0
19 verbose npm  v6.14.4
20 error code ELIFECYCLE
21 error errno 1
22 error zonemaster-gui@3.1.0 release: `npm run build && node scripts/create_release.js`
22 error Exit status 1
23 error Failed at the zonemaster-gui@3.1.0 release script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

I started again with new instructions:

```
matsd@u-1804-gui-bygg:~/git/zonemaster-gui$ npm --version
6.14.4
```

I created the directory that #110 will create, but it failed anyway.

```
matsd@u-1804-gui-bygg:~/git/zonemaster-gui$ npm run release

> zonemaster-gui@3.1.0 release /home/matsd/git/zonemaster-gui
> npm run build && node scripts/create_release.js

npm WARN invalid config loglevel="notice"

> zonemaster-gui@3.1.0 build /home/matsd/git/zonemaster-gui
> node scripts/faq.js && ng build --prod

An unhandled exception occurred: Job name "..getProjectMetadata" does not exist.
See "/tmp/ng-rZSZgH/angular-errors.log" for further details.
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! zonemaster-gui@3.1.0 build: `node scripts/faq.js && ng build --prod`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the zonemaster-gui@3.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/matsd/.npm/_logs/2020-04-01T13_49_00_736Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zonemaster-gui@3.1.0 release: `npm run build && node scripts/create_release.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the zonemaster-gui@3.1.0 release script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/matsd/.npm/_logs/2020-04-01T13_49_00_869Z-debug.log
```

And here is the log file:

```
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   '/home/matsd/.nvm/versions/node/v13.12.0/bin/node',
1 verbose cli   '/home/matsd/.nvm/versions/node/v13.12.0/bin/npm',
1 verbose cli   'run',
1 verbose cli   'release'
1 verbose cli ]
2 info using npm@6.14.4
3 info using node@v13.12.0
4 verbose run-script [ 'prerelease', 'release', 'postrelease' ]
5 info lifecycle zonemaster-gui@3.1.0~prerelease: zonemaster-gui@3.1.0
6 info lifecycle zonemaster-gui@3.1.0~release: zonemaster-gui@3.1.0
7 verbose lifecycle zonemaster-gui@3.1.0~release: unsafe-perm in lifecycle true
8 verbose lifecycle zonemaster-gui@3.1.0~release: PATH: /home/matsd/.nvm/versions/node/v13.12.0/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/matsd/git/zonemaster-gui/node_modules/.bin:/home/matsd/.nvm/versions/node/v13.12.0/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
9 verbose lifecycle zonemaster-gui@3.1.0~release: CWD: /home/matsd/git/zonemaster-gui
10 silly lifecycle zonemaster-gui@3.1.0~release: Args: [ '-c', 'npm run build && node scripts/create_release.js' ]
11 silly lifecycle zonemaster-gui@3.1.0~release: Returned: code: 1  signal: null
12 info lifecycle zonemaster-gui@3.1.0~release: Failed to exec release script
13 verbose stack Error: zonemaster-gui@3.1.0 release: `npm run build && node scripts/create_release.js`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/home/matsd/.nvm/versions/node/v13.12.0/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:315:20)
13 verbose stack     at ChildProcess.<anonymous> (/home/matsd/.nvm/versions/node/v13.12.0/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:315:20)
13 verbose stack     at maybeClose (internal/child_process.js:1026:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
14 verbose pkgid zonemaster-gui@3.1.0
15 verbose cwd /home/matsd/git/zonemaster-gui
16 verbose Linux 4.15.0-72-generic
17 verbose argv "/home/matsd/.nvm/versions/node/v13.12.0/bin/node" "/home/matsd/.nvm/versions/node/v13.12.0/bin/npm" "run" "release"
18 verbose node v13.12.0
19 verbose npm  v6.14.4
20 error code ELIFECYCLE
21 error errno 1
22 error zonemaster-gui@3.1.0 release: `npm run build && node scripts/create_release.js`
22 error Exit status 1
23 error Failed at the zonemaster-gui@3.1.0 release script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

--- a/src/assets/faqs/.gitkeep
+++ b/src/assets/faqs/.gitkeep
@@ -1,0 +1,1 @@
+# Empty gitkeep file to assure creation of public/js/lib directory by git

